### ISSUE TITLE
Warn about multiple commands on one line

### ISSUE THREAD

**Issue by l0b0** [koalaman/shellcheck#2592]
#### For new checks and feature suggestions
- [x] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/usr/bin/env bash
rm foo;echo bar;find baz
```

#### Here's what shellcheck currently says:

> No issues detected!

#### Here's what I wanted or expected to see:

> Split multiple commands into separate lines for clarity.

[Rationale](https://paperless.blog/bash-lints-beyond-shellcheck): Consistently lining commands up vertically makes it easier to understand the context of each command. If two commands belong “strongly” together, why not put them in a function to make that grouping explicit?

**Comment by koalaman** [comment]
I agree, but I'll leave this one up to `shfmt` which already does a better job at this. ShellCheck's main focus is issues affecting correctness.
